Added dns provider for ipv64.net

romanlum opened the pull request:
This PR adds support for the dns api of ipv64.net

romanlum commented:
Added issue for support
https://github.com/acmesh-official/acme.sh/issues/4419

romanlum commented:
Pipeline added.
The docker workflow fails although the certificates are created correctly.
Can you have a look if there is a problem with your workflow?

romanlum commented:
Thanks for approving the pull request.
Is anything more todo before merging the PR?

When should i create the WIKI article for ipv64.net dns api?

Bergruebe commented:
It would be great, if this PR could be merged.
Can I help anywhere?

Neilpang commented:
The dns api test must be passing.

https://github.com/romanlum/acme.sh/actions/runs/3581360113

romanlum commented:
I updated the branch to match the current dev and fixed the dns api tests
Now all pipelines are green :)

https://github.com/romanlum/acme.sh/actions/runs/4053033990
https://github.com/romanlum/acme.sh/actions/runs/4053033971
https://github.com/romanlum/acme.sh/actions/runs/4053033969

romanlum commented:
@Neilpang 
Would be great if you could merge the pull request :)
Thanks

Neilpang commented:
can you please add the usage here:  https://github.com/acmesh-official/acme.sh/wiki/dnsapi2

Neilpang commented on dnsapi/dns_ipv64.sh:
```diff
@@ -0,0 +1,157 @@
+  _debug _domain "$_domain"
+
+  # convert to lower case
+  _domain="$(echo "$_domain" | tr '[:upper:]' '[:lower:]')"
```
use the function "_upper_case()" and "_lower_case"

romanlum replied:
Changed code to use the lower_case function

romanlum commented:
> can you please add the usage here: https://github.com/acmesh-official/acme.sh/wiki/dnsapi2

How can I add this. I am not able to edit the page.

romanlum commented:
Code changed all pipelines are green :)
https://github.com/romanlum/acme.sh/actions/runs/4269752746
https://github.com/romanlum/acme.sh/actions/runs/4269752743
https://github.com/romanlum/acme.sh/actions/runs/4269752741

Neilpang commented:
>> How can I add this. I am not able to edit the page.

You can edit it now

romanlum commented:
updated https://github.com/acmesh-official/acme.sh/wiki/dnsapi2 and added ipv64 as provider 155
